core/templates/pages/exploration-editor-page/services/learner-answer-details-data.service.ts

```diff
@@ -17,81 +17,101 @@
  * any learner answer info.
  */
 
-require('pages/exploration-editor-page/services/exploration-data.service.ts');
-require('domain/utilities/url-interpolation.service.ts');
-require('domain/statistics/LearnerAnswerInfoObjectFactory.ts');
-require('domain/statistics/LearnerAnswerDetailsObjectFactory.ts');
+import {ExplorationDataService} from
+  './exploration-data.service';
+import {UrlInterpolationService} from
+  '../../../domain/utilities/url-interpolation.service';
+import {LearnerAnswerInfo} from
+  '../../../domain/statistics/LearnerAnswerInfoObjectFactory';
+import {LearnerAnswerDetails} from
+  '../../../domain/statistics/LearnerAnswerDetailsObjectFactory';
```

Comment:
Use absolute imports, you can use the same module to import from as it was present in the previous code. So something like: 
```
import {LearnerAnswerDetails} from
  '../../../domain/statistics/LearnerAnswerDetailsObjectFactory';
```

would become 
```
import {LearnerAnswerDetails} from
  'domain/statistics/LearnerAnswerDetailsObjectFactory';
```
as previously it was 
```
require('domain/statistics/LearnerAnswerDetailsObjectFactory.ts');
```

Reply:
Done